### PR TITLE
Fixes for spanish translations

### PR DESCRIPTION
**Pull request by drewpotter**
This is the initial work towards fixing all of the missing translations for the Spanish translation and a few other fixes.

### CODE REVIEW

File: app/assets/javascripts/survey.coffee.erb [hunk 1]
```diff
@@ -1,11 +1,22 @@
 # Place all the behaviors and hooks related to the matching controller here.
 # All this logic will automatically be available in application.js.
 # You can use CoffeeScript in this file: http://coffeescript.org/
+root = exports ? this
```
Comment (spencerldixon) [inline app/assets/javascripts/survey.coffee.erb]: Is this a ternary statement or some coffeescript thing? If so I think we need to be explicit about the false condition too

File: app/views/layouts/application.html.erb [hunk 2]
```diff
@@ -12,7 +12,7 @@
   </head>
 
   <body data-env="<%= Rails.env %>">
-    <div class="container">
+    <div class="container" id="container" data-lang="<%= I18n.locale %>">
```
Comment (spencerldixon) [inline app/views/layouts/application.html.erb]: I cant find where we're using this ID?

**Review by spencerldixon — CHANGES_REQUESTED** [review]

File: app/views/layouts/application.html.erb (hunk 2 above)
Comment (drewpotter) [reply app/views/layouts/application.html.erb]: Many thanks I will remove this!

File: app/assets/javascripts/survey.coffee.erb (hunk 1 above)
Comment (drewpotter) [reply app/assets/javascripts/survey.coffee.erb]: I have removed this unnecessary code thanks!

**Comment by drewpotter** [comment]
All done @spencerldixon !

**Review by spencerldixon — APPROVED** [review]

**Pull request merged**